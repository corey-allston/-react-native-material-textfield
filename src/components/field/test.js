import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import TextField from '.';

/* eslint-env jest */

it('renders text field', () => {
  let field = renderer
    .create(<TextField label='test' />)
    .toJSON();

  expect(field)
    .toMatchSnapshot();
});
