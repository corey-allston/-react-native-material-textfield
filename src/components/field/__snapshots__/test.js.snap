// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders text field 1`] = `
<View
  onResponderRelease={[Function]}
  onStartShouldSetResponder={[Function]}
>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
        "borderBottomColor": "rgba(0, 0, 0, 0.38)",
        "borderBottomWidth": 0.5,
        "height": 64,
        "paddingBottom": 8,
        "paddingTop": 32,
      }
    }
  >
    <View
      style={
        Object {
          "position": "absolute",
          "top": 36,
        }
      }
    >
      <Text
        accessible={true}
        active={false}
        allowFontScaling={true}
        animationDuration={225}
        disabled={false}
        ellipsizeMode="tail"
        errored={false}
        focused={false}
        numberOfLines={1}
        style={
          Object {
            "color": "rgba(0, 0, 0, 0.38)",
            "fontSize": 16,
          }
        }
      >
        test
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <TextInput
        autoCapitalize="sentences"
        blurOnSubmit={true}
        disableFullscreenUI={true}
        editable={true}
        onBlur={[Function]}
        onChange={[Function]}
        onChangeText={[Function]}
        onContentSizeChange={[Function]}
        onFocus={[Function]}
        selectionColor="rgb(0, 145, 234)"
        style={
          Array [
            Object {
              "flex": 1,
              "margin": 0,
              "padding": 0,
              "top": 2,
            },
            Object {
              "color": "rgba(0, 0, 0, .87)",
              "fontSize": 16,
              "height": 24,
            },
            undefined,
          ]
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "height": 8,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "paddingVertical": 4,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "rgb(213, 0, 0)",
              "fontSize": 0,
              "opacity": 0,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "paddingVertical": 4,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "rgba(0, 0, 0, .38)",
              "fontSize": 12,
              "opacity": 1,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
